package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "api.kitchen.support",
+  "version": "0.0.1",
+  "description": "Kitchen Support's RESTful API backend.",
+  "author": "Garrick Buckley <gbuckle@purdue.edu>",
+  "contributors": [
+    {
+      "name": "Garrick Buckley",
+      "email": "gbuckle@purdue.edu"
+    },
+    {
+      "name": "Jack Horton",
+      "email": "horton7@purdue.edu"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kitchensupport/api.git"
+  },
+  "dependencies": {
+    "express": "*",
+    "browserify": "*",
+    "gulp-ruby-sass": "*",
```

Comment:
any reason for ruby sass rather than libsass? https://www.npmjs.com/package/gulp-sass